src/window/window_wrapper.rs

```diff
@@ -215,6 +227,12 @@ impl WinitWindowWrapper {
     pub fn handle_focus_gained(&mut self) {
         EVENT_AGGREGATOR.send(UiCommand::Parallel(ParallelCommand::FocusGained));
         REDRAW_SCHEDULER.queue_next_frame();
+
+        // Got focus back after being minimized previously
+        if self.is_minimized {
+            EVENT_AGGREGATOR.send(UiCommand::Serial(SerialCommand::Keyboard("<NOP>".into())));
```

Comment:
It would be good to have a comment here telling that `<NOP>` triggers the `VimResume` autocommand

Reply:
Yes, done!